index.ts

```diff
@@ -149,7 +149,7 @@ addTests('isLabelList', [
 
 export const isMilestone = (url: URL | HTMLAnchorElement | Location = location): boolean => /^milestone\/\d+/.test(getRepo(url)?.path!);
 addTests('isMilestone', [
-	'https://github.com/sindresorhus/refined-github/milestone/12',
+	'https://github.com/sindresorhus/refined-github/milestones/12',
```

Comment:
`milestone` is right, you can see: https://github.com/kubernetes/kubernetes/milestone/56

Reply:
@kang8 Oops, I thought issue page with milestone filter is milestones page.
I'll change this url as you suggenstioned. Thank you!

Reply:
You can check the url on example is 404 page.
https://github.com/refined-github/refined-githu/milesone/12

Reply:
Out of conversation, why GH is using resources path messy?
for example, issue's path is https://github.com/refined-github/refined-github/issues/6000 but pr is https://github.com/refined-github/refined-github/pulls/100

Reply:
> why GH is using resources path messy?

Consistency is hard on large projects 😃